test/stores/widgets/WidgetPermissionStore-test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { mocked } from "jest-mock";
+import { MatrixClient } from "matrix-js-sdk/src/matrix";
+import { Widget, WidgetKind } from "matrix-widget-api";
+
+import { OIDCState, WidgetPermissionStore } from "../../../src/stores/widgets/WidgetPermissionStore";
+import SettingsStore from "../../../src/settings/SettingsStore";
+import { TestSdkContext } from "../../TestSdkContext";
+import { SettingLevel } from "../../../src/settings/SettingLevel";
+
+jest.mock("../../../src/settings/SettingsStore", () => ({
```

Comment:
You can just do `jest.mock("../../../src/settings/SettingsStore");` here.
Jest mocks everything in a module by default.